scripts/scil_convert_json_to_xlsx.py

```diff
@@ -0,0 +1,416 @@
+                            means[bundle_id * len(labels) + label_id,
+                                  metric_id] =\
+                                label_stat['mean']
+                            stddev[bundle_id * len(labels) + label_id,
+                                   metric_id] =\
+                                label_stat['std']
+
+    column_names = []
+    for bundles_name in bundles:
+        column_names.extend(["{}_{}".format(bundles_name, label)
+                             for label in labels])
+
+    dataframes = []
+    df_names = []
+    index = ['Population']
+    for metric_id, metric_name in enumerate(metrics):
+        dataframes.append(pd.DataFrame(data=np.array([means[:, metric_id]]),
+                                       index=index,
+                                       columns=column_names))
+        df_names.append(metric_name + "_mean")
+
+        dataframes.append(pd.DataFrame(data=np.array([stddev[:, metric_id]]),
+                                       index=index,
+                                       columns=column_names))
+        df_names.append(metric_name + "_std")
+
+    return dataframes, df_names
+
+
+def _create_xlsx_from_json(json_path, xlsx_path,
+                           sort_subs=True, sort_bundles=True,
+                           ignored_bundles_fpath=None,
+                           stats_over_population=False):
+    with open(json_path, 'r') as json_file:
+        stats = json.load(json_file)
+
+    subs = list(stats.keys())
+    if sort_subs:
+        subs.sort()
+
+    bundle_names = _get_all_bundle_names(stats)
+    if sort_bundles:
+        bundle_names.sort()
+
+    if ignored_bundles_fpath is not None:
+        with open(ignored_bundles_fpath, 'r') as f:
+            bundles_to_ignore = [l.strip() for l in f]
+        bundle_names = filter(lambda name: name not in bundles_to_ignore,
+                              bundle_names)
+
+    cur_stats_func = _get_stats_parse_function(stats, stats_over_population)
+
+    dataframes, df_names = cur_stats_func(stats, subs, bundle_names)
+
+    if len(dataframes):
+        _write_dataframes(dataframes, df_names, xlsx_path)
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter,
+                                description=__doc__)
+
+    p.add_argument('in_json',
+                   help='File containing the json stats.')
+
+    p.add_argument('out_xlsx',
+                   help='Output Excel file for the stats.')
+
+    p.add_argument('--no_sort_subs', action='store_false',
+                   help='If set, subjects won\'t be sorted alphabetically.')
+
+    p.add_argument('--no_sort_bundles', action='store_false',
+                   help='If set, bundles won\'t be sorted alphabetically.')
+
+    p.add_argument('--ignore_bundles', metavar='FILE',
+                   help='Path to a text file containing a list of bundles '
+                        'to ignore.')
```

Comment:
(.txt) 

One bundle per line? I never used this option? What is the format? Can you provide a bit more info